### PR TITLE
Added support of formtarget attribute in yii.activeForm.js [skip ci]

### LINKED ISSUE #10322: ActiveForm prevents button to open in new window (formtarget="_blank")
The **formtarget** directive is not working when the button is part of an ActiveFormorm.

```php
// This example does not work.  It submits the form data to the current window.
<?php $form = ActiveForm::begin(['method' => 'get']); ?>
        First name: <input type="text" name="fname"><br>
        Last name: <input type="text" name="lname"><br>
        <button type="submit" >Submit</button>
        <button type="submit" formtarget="_blank">Submit to a new window</button>
    <?php ActiveForm::end(); ?> 
```

```php
// This example works well. It submits the form data to a new window.
 <form id="w0"  method="get"> 
        First name: <input type="text" name="fname"><br>
        Last name: <input type="text" name="lname"><br>
        <button type="submit" >Submit</button>
        <button type="submit" formtarget="_blank">Submit to a new window</button>
  </form> 
```

### PR DESCRIPTION
**Pull request by AnatolyRugalev**
Fixes #10322

### CODE REVIEW

File: framework/assets/yii.activeForm.js [hunk 1]
```diff
@@ -201,7 +201,8 @@
                     settings: settings,
                     attributes: attributes,
                     submitting: false,
-                    validated: false
+                    validated: false,
+                    target: data.target === undefined ? null : data.target
```
Comment (SilverFire) [inline framework/assets/yii.activeForm.js]: may be simplified to `data.target || null`

**Comment by AnatolyRugalev** [comment]
@SilverFire, fixed

File: framework/assets/yii.activeForm.js [hunk 2]
```diff
@@ -201,7 +201,8 @@
                     settings: settings,
                     attributes: attributes,
                     submitting: false,
-                    validated: false
+                    validated: false,
+                    target: data.target || null
```
Comment (SilverFire) [inline framework/assets/yii.activeForm.js]: where does the `data` come from?

**Comment by SilverFire** [comment]
@AnatolyRugalev could you update from the upstream, please?

**Comment by AnatolyRugalev** [comment]
@SilverFire merged

**Comment by SilverFire** [comment]
Merged, thank you!

**Comment by demogorgorn** [comment]
After this fix the submit button in active form works only on the second time.
In JS console: ReferenceError: $button is not defined

**Comment by SilverFire** [comment]
@AnatolyRugalev reverted this PR https://github.com/yiisoft/yii2/commit/5686a9ddebad3d8b6f1d041d8c45403937c7de09

It seems, something gone wrong when merging with upstream and I've overseen this problem :(

**Comment by SilverFire** [comment]
@AnatolyRugalev could you fix it, please?

**Comment by AnatolyRugalev** [comment]
@SilverFire sure, will fix it today

**Comment by SilverFire** [comment]
Submit is a s new PR, please

**Comment by AnatolyRugalev** [comment]
@SilverFire, [done](https://github.com/yiisoft/yii2/pull/11410)

**Comment by SilverFire** [comment]
Resolved by https://github.com/yiisoft/yii2/commit/fbedb8e750dbb752e43af718f4692c7d5d5e43d7

**Pull request merged**